source/client/options_impl.cc

```diff
@@ -166,18 +166,42 @@ OptionsImpl::OptionsImpl(int argc, const char* const* argv) {
   uri_ = uri.getValue();
   TCLAP_SET_IF_SPECIFIED(h2, h2_);
   TCLAP_SET_IF_SPECIFIED(concurrency, concurrency_);
-  TCLAP_SET_IF_SPECIFIED(verbosity, verbosity_);
-  TCLAP_SET_IF_SPECIFIED(output_format, output_format_);
+  // TODO(oschaaf): is there a generic way to set these enum values?
+  if (verbosity.isSet()) {
+    std::string upper_cased = verbosity.getValue();
+    absl::AsciiStrToUpper(&upper_cased);
+    ASSERT(nighthawk::client::Verbosity::VerbosityOptions_Parse(upper_cased, &verbosity_));
+  }
+  if (output_format.isSet()) {
+    std::string upper_cased = output_format.getValue();
+    absl::AsciiStrToUpper(&upper_cased);
+    ASSERT(
+        nighthawk::client::OutputFormat::OutputFormatOptions_Parse(upper_cased, &output_format_));
+  }
   TCLAP_SET_IF_SPECIFIED(prefetch_connections, prefetch_connections_);
   TCLAP_SET_IF_SPECIFIED(burst_size, burst_size_);
-  TCLAP_SET_IF_SPECIFIED(address_family, address_family_);
-  TCLAP_SET_IF_SPECIFIED(request_method, request_method_);
+  if (address_family.isSet()) {
+    std::string upper_cased = address_family.getValue();
+    absl::AsciiStrToUpper(&upper_cased);
+    ASSERT(nighthawk::client::AddressFamily::AddressFamilyOptions_Parse(upper_cased,
```

Comment:
Can't use `ASSERT` here, as it gets removed in opt builds, use `RELEASE_ASSERT` (or preferably exceptions) instead.

Reply:
I will change these to `RELEASE_ASSERT`. With respect to exception, we shouldn't be able to get here at all, because the validation earlier should prevent that. Given that, do you still prefer the exception?

Reply:
`RELEASE_ASSERT` is good.

Reply:
Done